src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -38,6 +38,36 @@
 
 namespace triton { namespace perfanalyzer {
 
+bool
+IsPositiveInteger(const std::string& str)
```

Comment:
It looks like some validations use `IsPositiveInteger`/`IsNonNegativeInteger`, and others use `std::stoi`/`std::stoull`, etc. I think all of them might as well just use the `std::` ones instead of these functions. No need to reinvent the wheel

Reply:
But then shouldn't `IsPositiveInteger`/`IsNonNegativeInteger` be used everywhere?

Reply:
Changed to using the custom checks and removing try/catch